static/src/stylesheets/module/journalism/_audio-flagship-tag-page.scss

```diff
@@ -181,7 +187,7 @@
         display: block;
         margin: 0;
 
-        @include mq($from: mobile, $until: phablet) {
+        @include mq($until: phablet) {
             display: none;
```

Comment:
I removed the `$from` here and below to persist the styles at smaller breakpoints than `mobile`